spec.html

```diff
@@ -23133,13 +23122,10 @@ <h1>Cyclic Module Records</h1>
 
         <emu-clause id="sec-moduledeclarationlinking" oldids="sec-moduledeclarationinstantiation">
           <h1>Link ( ) Concrete Method</h1>
-          <p>The Link concrete method of a Cyclic Module Record implements the corresponding Module Record abstract method.</p>
-          <p>On success, Link transitions this module's [[Status]] from ~unlinked~ to ~linked~. On failure, an exception is thrown and this module's [[Status]] remains ~unlinked~.</p>
-          <p>This abstract method performs the following steps (most of the work is done by the auxiliary function InnerModuleLinking):</p>
+          <p>The Link concrete method of a Cyclic Module Record _module_ takes no arguments. On success, Link transitions this module's [[Status]] from ~unlinked~ to ~linked~. On failure, an exception is thrown and this module's [[Status]] remains ~unlinked~. (Most of the work is done by the auxiliary function InnerModuleLinking.) It performs the following steps when called:</p>
```

Comment:
```suggestion
          <p>The Link concrete method of a Cyclic Module Record _module_ takes no arguments. On success, Link transitions this module's [[Status]] from ~unlinked~ to ~linked~. On failure, an exception is thrown and this module's [[Status]] remains ~unlinked~. Most of the work is done by the auxiliary function InnerModuleLinking. It performs the following steps when called:</p>
```

Reply:
I kept the parentheses from the status quo because if you take them out, it sounds like the subsequent "It" refers to `InnerModuleLinking`. I'd be more inclined to drop the sentence than de-parenthesize it.